### ISSUE TITLE
PDO unable to parse without ExprType

### ISSUE THREAD

**Issue by kevinresol** [haxetink/tink_sql#111]
I think it should use [getColumnMeta](https://www.php.net/manual/en/pdostatement.getcolumnmeta.php) to determine the type returned from the database.

Enable `ProcedureTest` to reproduce.

**Comment by benmerckx** [comment]
Yeah, the last changes basically went from getting type at runtime from the driver to fully relying on type information. What if we add type to procedure too?
